app/Actions/CustomizeConfigRuntime.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Actions;
+
+use App\Facades\Options;
+use App\Support\BaseAction;
+
+class CustomizeConfigRuntime extends BaseAction
+{
+    /**
+     * Customize the configuration in runtime.
+     *
+     * @return void
+     */
+    public function __invoke(): void
+    {
+        $option = $this->console
+            ->menu('Change configuration value', Options::interactiveMenuOptions())
+            ->open();
+
+        $nothingChanged = ['Nothing was changed. Ready to go?', 'alert'];
+        [$message, $level ] = $nothingChanged;
+
+        if ($option !== null) {
+            $selection = Options::perform($option, $this->console);
+
+            if ($selection->value() !== null) {
+                $message    = "You have set the configuration [{$option}] to: {$selection->value()}";
+                $level      = 'info';
+            } else {
+                $message    = $selection->message();
+                $level      = $selection->messageLevel();
+            }
+        }
+
+        $this->console->initialScreen($message, $level);
```

Comment:
This whole method is a bit tough to follow. Let's try this instead:

```php
// get option

if (option=== null )
this->console->initialScreen(values for nothing changed)
return
}

// .. now go forward assuming we have an option

```

Just drops a bit of the complexity